hapi/hedera-protobuf-java-api/src/main/proto/fees/fee_schedule.proto

```diff
@@ -94,6 +94,8 @@ enum Extra {
     ALLOWANCES=14;
     AIRDROPS=15;
     HOOKS=16;
+    CONSENSUS_CREATE_TOPIC_WITH_CUSTOM_FEE=17;
```

Comment:
nit: These need not be the same names as `SubTypes`, if we can reduce their length